artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/AddressControl.java

```diff
@@ -135,4 +135,34 @@ String sendMessage(@Parameter(name = "headers", desc = "The headers to add to th
                       @Parameter(name = "durable", desc = "Whether the message is durable") boolean durable,
                       @Parameter(name = "user", desc = "The user to authenticate with") String user,
                       @Parameter(name = "password", desc = "The users password to authenticate with") String password) throws Exception;
+
+   /**
+    * Pauses all the consumers of this address.Messages are no longer delivered to all its queues consumers.
```

Comment:
Ditto

Reply:
sorry

Reply:
Same issue as other comments you just fixed